source/kraken/make_disruption_from_chaos.cpp

```diff
@@ -36,6 +36,20 @@ namespace bt = boost::posix_time;
 
 namespace navitia {
 
+static void update_property(nt::disruption::Property& property, const chaos::Property& chaos_property) {
```

Comment:
Not sure it's worth it (requires a new PR on chaos-proto) : chaos::`Disruption`Property would be clearer (property is way too generic).

Reply:
PR incoming